jobs.go

```diff
@@ -104,7 +104,7 @@ type Bridge struct {
 	Tag                bool   `json:"tag"`
 	WebURL             string `json:"web_url"`
 	User               *User  `json:"user"`
-	DownstreamPipeline struct {
+	DownstreamPipeline *struct {
```

Comment:
If we make this a pointer, we should make a named struct for this (to keep the code consistent). Make define:

```
type PipelineInfo struct {
	ID        int        `json:"id"`
	Ref       string     `json:"ref"`
	Sha       string     `json:"sha"`
	Status    string     `json:"status"`
	CreatedAt *time.Time `json:"created_at"`
	UpdateAt  *time.Time `json:"updated_at"`
	WebURL    string     `json:"web_url"`
}
```

Which can then be used for the `Pipeline` and `DownstreamPipeline` fields?

Reply:
Thats a great suggestion. I have pushed an additional commit to make use of `PipelineInfo` in pipeline.go

Reply:
Cool... I even forgot that it already existed 😂 Could you also use the same struct for the `Pipeline` field to keep it consistent? Thanks!

Reply:
Added another commit do replace `Pipeline` field, but I kept it as value instead of switching it to pointer.